Cancelamento de NFe

BiancaBartolomei opened the pull request:
Atividade referente a  Fiscal - Fluxo do Cancelamento da NFe

BiancaBartolomei committed:
[WIP]Cancelamento de NFe

mileo commented on sped_nfe/wizards/wizard_nfe_motivo_cancelamento.py:
```diff
@@ -0,0 +1,35 @@
+from odoo.exceptions import Warning as UserError
+
+
+class FinanLancamento(models.TransientModel):
```
???????????

hendrixcosta committed:
[FIX] Chamando wizard direto pela action

hendrixcosta committed:
[FIX] Refatoração da view e nomes

hendrixcosta committed:
Update inherited_sped_documento_emissao_nfe_view.xml

hendrixcosta committed:
Update inherited_sped_documento.py

BiancaBartolomei commented:
Funciona corretamente

mileo commented on sped_nfe/wizards/wizard_nfe_motivo_cancelamento.py:
```diff
@@ -0,0 +1,33 @@
+        self.ensure_one()
+
+        if len(self.motivo_cancelamento) < 10:
+            raise UserError("A justificativa deve ter mais de 10 caracteres.")
```
Se eu não me engano o cancelamento tb tem um tamanho máximo. Seria interessante colocar este limite no campo limit do text/char.

hendrixcosta replied:
@BiancaBartolomei

hendrixcosta committed:
[ADD] campo reference no sped.documento para identificar qual a origem do documento (#493)

[ADD] Informação da sale_order na visao lista do sped.documento

Update sped_documento.py

[FIX] Remover campo obrigatório

lfdivino committed:
[ADD] Informações adicionais no CFe através do template no operação fiscal (#495)

mileo committed:
[IMP] Cancelamento e inutilização de NF-E/NFS-E

mileo committed:
[IMP] Cancelamento e inutilização de CF-E

hendrixcosta commented:
Cancelou a nota corretamente.
![image](https://user-images.githubusercontent.com/7250580/35460199-4bee37f0-02ca-11e8-936d-491b547971ec.png)
